dev-lang/rust/rust-1.51.0-r2.ebuild

```diff
@@ -41,7 +41,7 @@ LLVM_TARGET_USEDEPS=${ALL_LLVM_TARGETS[@]/%/(-)?}
 
 LICENSE="|| ( MIT Apache-2.0 ) BSD-1 BSD-2 BSD-4 UoI-NCSA"
 
-IUSE="clippy cpu_flags_x86_sse2 debug doc miri nightly parallel-compiler rls rustfmt system-bootstrap system-llvm test wasm ${ALL_LLVM_TARGETS[*]}"
+IUSE="clippy cpu_flags_x86_sse2 debug doc miri multilib nightly parallel-compiler rls rustfmt system-bootstrap system-llvm test wasm ${ALL_LLVM_TARGETS[*]}"
```

Comment:
Is there a very good reason to support non-multilib builds in the first place?

Reply:
I don't think so, aside from wasting time building unusable code in `no-multilib` profiles.  I just tried to make the interface consistent with glibc, since the behavior is to build more std libs for additional ABIs.

Reply:
Well, I don't think that there's anything stopping you from controlling multilib via profile without extra flags. In fact, `get_all_abis` should be doing already, I think.